src/client/app/effects/snippet.ts

```diff
@@ -365,6 +403,15 @@ export class SnippetEffects {
         snippet.gist = rawSnippet.gist;
         snippet.gistOwnerId = rawSnippet.gistOwnerId;
 
+        if (snippet.gist && this._github.profile && this._github.profile.login === snippet.gistOwnerId) {
```

Comment:
I like that you have the snippet-trusting (`trustedSnippetsKey`) in a helper.  But in that case, why not have the logic below execute there too?